Doc/library/sqlite3.rst

```diff
@@ -1562,6 +1539,8 @@ Cursor objects
 
       Use :meth:`executescript` to execute multiple SQL statements.
 
+      :meth:`!execute` only accepts UTF-8 encoded strings.
```

Comment:
@serhiy-storchaka, do you oppose this change?

Reply:
It is technically incorrect. I am not sure what wording would be better here and whether a special addition is needed at all, because for functions have the same requirement, but it is not documented, and in general you cannot know whether surrogate characters are acceptable until you try.

It is not actually clear to what the note refers. Is it only *sql* or *parameters*? Parameters can contain not only strings, but numbers, bytes and custom data. Strings must be UTF-8 encodable, but bytes can be not only UTF-8 encoded.

Reply:
*sql* is documented as being a `str` instance; not a `bytes` instance. There is no need to complicate the docs with encoding details here.